subjects/java/checkpoints/area-calculator/README.md

```diff
@@ -0,0 +1,82 @@
+## Area Calculator
+
+### Instructions
+
+Create a class `AreaCalculator` that has an overloaded method `calculate` to calculate the area of different shapes. The method should be able to calculate the area for:
+
+- A circle (given the radius)
+- A rectangle (given the width and height)
+- A triangle (given the base and height)
+- A square (given the side length)
+
+#### Formulas for area calculations:
+
+- Circle: Area = 𝜋 × radius²
+
+- Rectangle: Area = width × height
+
+- Triangle: Area = ½ × base × height
+
+- Square: Area = side²
+
+### Expected Class
+
+```java
+public class AreaCalculator {
+    public double calculate(double radius) {
+        // calculate area of circle
+    }
+
+    public double calculate(double width, double height) {
+        // calculate area of rectangle
+    }
+
+    public double calculate(double base, double height, boolean isTriangle) {
```

Comment:
What should happen if `isTriangle` is `false`?

Reply:
the same for isSquare